src/lsm/manifest.zig

```diff
@@ -370,8 +380,6 @@ pub fn ManifestType(comptime Table: type, comptime Storage: type) type {
         }
 
         /// Returns the next table in the range, after `key_exclusive` if provided.
-        ///
-        /// * The table returned is visible to `snapshot`.
```

Comment:
`ManifestLevel.next_table` says it's visible to the snapshot. Is it no longer the case for this function?